### ISSUE TITLE
Cannot select multiple commits in history to squash

### ISSUE THREAD

**Issue by mposch-Z** [desktop/desktop#17599]
### The problem

Cannot select multiple commits in history to squash.  Currently running V2.5.6.  This is standard OS behavior and according to many stack overflow posts/github desktop threads it seems users are able to select numerous commits in the history tab and squash into a single commit.  This feature does not appear to be working on the latest version, or the feature has been removed?

### Release version

2.5.6

### Operating system

Windows 10

### Steps to reproduce the behavior

Go To history tab
Try and control click OR shift click numerous commits in the history.  Ultimately only a single entry is ever highlighted.  Also cannot drag and drop as indicated here:
https://docs.github.com/en/desktop/managing-commits/squashing-commits-in-github-desktop

**Comment by sergiou87** [comment]
Hello @mposch-Z !

I'm afraid 2.5.6 is a really old version of the app that doesn't support squashing commits. We recommend you to install the latest version from https://desktop.github.com with the latest features and security fixes.

**Comment by mposch-Z** [comment]
Hello -
I updated to the latest version and was able to highlight numerous items in the history.  Followed the squash dialog but never received a confirmation of a successful squash.  Nothing is reflecting in the repo history as well (still shows ALL the commits I tried to squash into 1).  Now, when trying to re-perform the operation I can no longer select the squash option, although it is still shown:
***@***.***
